log-ballerina/natives.bal

```diff
@@ -159,7 +159,52 @@ isolated function stderrStream() returns handle = @java:FieldGet {
     'class: "java/lang/System"
 } external;
 
-isolated function printLogFmtExtern(LogRecord logRecord) returns string = @java:Method {'class: "org.ballerinalang.stdlib.log.Utils"} external;
+isolated function printLogFmtExtern(LogRecord logRecord) returns string {
```

Comment:
```suggestion
isolated function printLogFmt(LogRecord logRecord) returns string {
```

Reply:
shall we remove extern part from the function name.

Reply:
Updated in ce2e747